torch/optim/lr_scheduler.py

```diff
@@ -187,8 +185,9 @@ def __exit__(self, type, value, traceback):
 
 
 class LambdaLR(LRScheduler):
-    """Sets the learning rate of each parameter group to the initial lr
-    times a given function. When last_epoch=-1, sets initial lr as lr.
+    """Sets the learning rate of each parameter group to the initial lr times a given function.
+
+    When last_epoch=-1, sets initial lr as lr.
```

Comment:
```suggestion
    When last_epoch=-1, set initial lr as lr.
```
here too

Reply:
maybe not, since this is a class..

Reply:
I also didn't touch them because they are in a class and not the summary line.